README.md

```diff
@@ -1,2 +1,24 @@
-# aem-webpack-example
-Best practice for setting up Webpack in AEM.
+# AEM Webpack Example
+
+This project demonstrates a proven practice for setting up Webpack in AEM. It provides a straightforward Webpack configuration and supports Babel and ESLint.
```

Comment:
I feel that we need to bring a little "branding" there. For instance list out 2 or 3 high level benefits of using this practice as well as mentioning Infield Digital in this first paragraph somehow.